### PR TITLE
doc: update router variable example (#2298)

### PR DESCRIPTION
**Pull request by diaosj**
### What this PR does / why we need it:
Update the `router-radixtree` document.

The former router example in this document use "==" as the default operator, which is not the current behavior.
This PR modifies the example and keeps it consistent with the `admin-api` document.

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible?

### CODE REVIEW

**Review by moonming — CHANGES_REQUESTED** [review]
We should keep the example of http header and cookie

**Review by nic-chen — APPROVED** [review]

**Review by membphis — APPROVED** [review]

File: doc/router-radixtree.md [hunk 1]
```diff
@@ -78,9 +78,11 @@ $ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f
 {
     "uri": "/index.html",
     "vars": [
-        ["http_host", "iresty.com"],
-        ["cookie__device_id", "a66f0cdc4ba2df8c096f74c9110163a9"],
-        ["arg_name", "jack"]
+        ["http_host", "==", "iresty.com"],
+        ["cookie__device_id", "==", "a66f0cdc4ba2df8c096f74c9110163a9"],
```
Comment (moonming) [inline doc/router-radixtree.md:82]: Two _?

Comment (juzhiyuan) [reply doc/router-radixtree.md:82]: in order to swprate keyword and custom namre?

Comment (moonming) [reply doc/router-radixtree.md:82]: I am not sure about this

Comment (moonming) [reply doc/router-radixtree.md:82]: it's a typo: http://nginx.org/en/docs/http/ngx_http_core_module.html#var_cookie_
only one `_`

**Review by moonming — CHANGES_REQUESTED** [review]

**Review by moonming — APPROVED** [review]

**Pull request merged**